src/printer.ts

```diff
@@ -1056,7 +1057,63 @@ export class PugPrinter {
 
 	private ['start-pipeless-text'](token: StartPipelessTextToken): string {
 		this.pipelessText = true;
-		return `\n${this.indentString.repeat(this.indentLevel)}`;
+
+		let result: string = `\n${this.indentString.repeat(this.indentLevel)}`;
+
+		if (this.previousToken?.type === 'dot') {
+			const lastTagToken: TagToken | undefined = previousTagToken(this.tokens, this.currentIndex);
+
+			let parser: BuiltInParserName | undefined;
+			switch (lastTagToken?.val) {
+				case 'script':
+					parser = 'babel';
+					break;
+				case 'style':
+					parser = 'css';
+					break;
+				default:
+					break;
+			}
+
+			if (parser) {
+				let index: number = this.currentIndex + 1;
+				let tok: Token | undefined = this.tokens[index];
+				let rawText: string = '';
+				while (tok && tok?.type !== 'end-pipeless-text') {
+					switch (tok.type) {
+						case 'text':
+							rawText += tok.val;
+							break;
+						case 'newline':
+							rawText += '\n';
+							break;
+						default:
+							logger.warn('Unhandled token for pipeless script tag:', JSON.stringify(tok));
+							break;
+					}
+
+					index++;
+					tok = this.tokens[index];
+				}
+
+				result = format(rawText, { parser, ...this.codeInterpolationOptions });
+				const indentString: string = this.indentString.repeat(this.indentLevel + 1);
+				result = `\n${result
+					.split('\n')
+					.map((line) => indentString + line)
+					.join('\n')}`.trimRight();
```

Comment:
```suggestion
				result = `\n${result
				    .trimRight()
					.split('\n')
					.map((line) => indentString + line)
					.join('\n')}`;
```
If we remove the newline first, then we don't have to prefix it with the indentation before removing it (minimal performance gain).

Reply:
Ah okay, but I would prefer to write:
```ts
result = format(rawText, { parser, ...this.codeInterpolationOptions });
result = result.trimRight();
const indentString: string = this.indentString.repeat(this.indentLevel + 1);
result = result.split('\n').map((line) => indentString + line).join('\n');
result = `\n${result}`;
```
Should be the same, but much more readable, right?